packages/api/src/index.ts

```diff
@@ -30,7 +30,7 @@ export type Actions = { [key: string]: Action };
 // this represents valid currencies to check prices against on coingecko
 // see: https://www.coingecko.com/api/documentations/v3#/asset_platforms/get_asset_platforms
 export type CurrencySymbol = "usd";
-export type PriceSample = [timestamp: number, price: string];
```

Comment:
Why drop the names?

Reply:
i was getting a build error for some reason. going back to see if i still get it

Reply:
looks like it does build with this in now